src/publicKeyProvider/CachingKeyProvider.ts

```diff
@@ -0,0 +1,36 @@
+import { ConnectJwt } from '../types';
```

Comment:
CachingKeyProvider is an opt-in implementation that can be used by consumers of the library. It's not possible to have it used anywhere here because it needs an underlying storage which must be provided when creating an instance of this class.